src/dbus/common/types.hpp

```diff
@@ -57,17 +57,10 @@ enum DeviceRole
 struct ActiveScanResult
 {
     uint64_t             mExtAddress;    ///< IEEE 802.15.4 Extended Address
-    std::string          mNetworkName;   ///< Thread Network Name
-    uint64_t             mExtendedPanId; ///< Thread Extended PAN ID
-    std::vector<uint8_t> mSteeringData;  ///< Steering Data
     uint16_t             mPanId;         ///< IEEE 802.15.4 PAN ID
-    uint16_t             mJoinerUdpPort; ///< Joiner UDP Port
     uint8_t              mChannel;       ///< IEEE 802.15.4 Channel
     int8_t               mRssi;          ///< RSSI (dBm)
     uint8_t              mLqi;           ///< LQI
-    uint8_t              mVersion;       ///< Version
-    bool                 mIsNative;      ///< Native Commissioner flag
-    bool                 mDiscover;      ///< Result from MLE Discovery
 };
```

Comment:
I wonder if we should remove these fields from the struct. It seems they are still present in `otActiveScanResult`. @bukepo do you have any idea whether we should remove them?

Reply:
I agree with you. We should keep these fields until they are not available in OpenThread anymore.

Reply:
Gotcha, I can add all those values back to the structure. Do you want to keep the introspect as it is or would you like to return the entire structure?

Reply:
I'd prefer update the introspect to return the entire structure.

Reply:
Fixed!